src/core/array.d

```diff
@@ -0,0 +1,61 @@
+module core.array;
+
+/++
+    Returns a static array constructed from `a`. The type of elements can be
+    specified implicitly (`int[2] a = staticArray(1,2)`) or explicitly
+    (`float[2] a = staticArray!float(1,2)`).
```

Comment:
Params + Returns

Reply:
i think this is too much boilerplate and doesn't add anything in this case; looking at https://dlang.org/phobos/std_array.html#array , functions where it's obvious from rest of ddoc don't do it either.